db/migrate/20220701205329_create_users.rb

```diff
@@ -9,7 +9,7 @@ def change
       t.text :description, :default => ""
       t.string :user_image
 
-      t.timestamps null: false
+      t.timestamps
```

Comment:
I decided to keep timestamps for every table, so keeping them from being null is preferable.